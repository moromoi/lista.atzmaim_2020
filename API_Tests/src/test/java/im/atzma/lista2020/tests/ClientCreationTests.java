package im.atzma.lista2020.tests;

import org.testng.annotations.Test;

import java.io.IOException;

public class ClientCreationTests extends TestBase {

    @Test
    public void createClient() throws IOException {
        for (int i = 0; i < 100; i++) {

            app.clientCreationHelper().createClient();
        }
       // app.clientCreationHelper().getClientList();
       // app.clientCreationHelper().deleteClient();

       app.clientCreationHelper().beforeCreation();
    }
}
