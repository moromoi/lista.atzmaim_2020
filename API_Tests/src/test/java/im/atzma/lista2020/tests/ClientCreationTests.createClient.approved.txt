Server=nginx
###Date##
Content-Type=application/json; charset=utf-8
Transfer-Encoding=chunked
Connection=keep-alive
Vary=Accept-Encoding
Set-##Cookie###
Set-##Cookie###
###X-Request-Id##
Strict-Transport-Security=max-age=604800
X-Content-Type-Options=nosniff
X-Robots-Tag=noindex
Content-Security-Policy=upgrade-insecure-requests
X-Powered-By=php/4.3.10-16
Content-Encoding=gzip

[
    {
        "##ID##
        "name": "Test_changeClientsData_before",
        "phone": null,
        "###profile_image##
        "address": "Israel, Rokah 18, Ramat Gan",
        "status": "status - before",
        "###last_appointment##
    }
]